src/main/java/pl/jakubowskir/timetable/trainer/controller/TrainerController.java

```diff
@@ -0,0 +1,54 @@
+package pl.jakubowskir.timetable.trainer.controller;
+
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import pl.jakubowskir.timetable.trainer.model.Trainee;
+import pl.jakubowskir.timetable.trainer.model.Trainer;
+import pl.jakubowskir.timetable.trainer.repository.TrainerRepository;
+
+
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/trainers")
+public class TrainerController {
+    private final TrainerService trainerService;
+    private final TrainerRepository trainerRepository;
+
+    @GetMapping("/api/v1/trainer")
+    public List<Trainer> getTrainers() {
+        return trainerService.getTrainers();
+    }
+
+    @PostMapping("/trainer")
+    public ResponseEntity<String> addTrainer(@RequestBody TrainerDto trainerDto) {
+        Trainer addedTrainer = trainerService.addTrainer(trainerDto);
+        if (addedTrainer != null) {
+            return ResponseEntity.ok("Dodano nowego trenera");
+        } else {
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body("Błąd dodawania trenera");
+        }
+    }
+
+    @PostMapping("/{trainerId}/trainee") // dodawanie podopiecznego do konkretnego trenera po ID
+    public ResponseEntity<String> addTraineeToTrainer(@PathVariable Long trainerId, @RequestBody Trainee trainee) {
```

Comment:
TraineeDto zamiast Trainee, bo nie powinniśmy przekazywać jako json całego obiektu (wymagało by to dużo ustawiania niepotrzebnego). Zrób podobnie jak w addTrainer. Dodatkowo, czy podopieczny ma na początku trenera? Może być bez .. Chyba tak. Więc mozna by było zrobić metodę dodającą podopiecznego do trenera i osobną do dodania podopiecznego. Wtedy byśmy przekazywali trainerId i traineeId